### ISSUE TITLE
Great job, some questions about the dataset and running demo.

### ISSUE THREAD

**Issue by booker-max** [WU-CVGL/Omni-Scene#4]
First and foremost, I want to express my sincere admiration for your excellent work. The way you ingeniously combined pixel-tracing and volume-based methods is truly impressive.

I have several questions regarding your work that I would greatly appreciate your insights on:

1. Could you please share the approximate size of the pre-processed dataset? This information would help me better plan my download strategy.

2. If I intend to use this as a tool or run demonstrations, would it be sufficient to download only the dataset_omniscene_part00 package? Or do you have any alternative suggestions that would allow me to experiment with and validate its effectiveness more efficiently?

3. Additionally, I'm planning to fine-tune the model using my own autonomous driving dataset. In this regard, I would greatly value your recommendations on:
Which modules should be frozen
Which modules should be trained with a lower learning rate
Any other best practices you might suggest for the fine-tuning process

Thank you very much for your time and consideration. I look forward to your response.

Best regards

**Comment by wswdx** [comment]
> First and foremost, I want to express my sincere admiration for your excellent work. The way you ingeniously combined pixel-tracing and volume-based methods is truly impressive.
> 
> I have several questions regarding your work that I would greatly appreciate your insights on:
> 
> 1. Could you please share the approximate size of the pre-processed dataset? This information would help me better plan my download strategy.
> 2. If I intend to use this as a tool or run demonstrations, would it be sufficient to download only the dataset_omniscene_part00 package? Or do you have any alternative suggestions that would allow me to experiment with and validate its effectiveness more efficiently?
> 3. Additionally, I'm planning to fine-tune the model using my own autonomous driving dataset. In this regard, I would greatly value your recommendations on:
>    Which modules should be frozen
>    Which modules should be trained with a lower learning rate
>    Any other best practices you might suggest for the fine-tuning process
> 
> Thank you very much for your time and consideration. I look forward to your response.
> 
> Best regards

Thanks for your admiration! About your concerns:
1. The approximate size of the pre-processed dataset is 600GB. You can download it via Onedrive or BaiduYunPan.
2. Thanks! You really inspired me! I will provide a small subset of our pre-processed data that is easy to download for convenient demo running. But for now, you can't only download dataset_omniscene_part00 to extract files validly.
3. I have not tried this before. But I can give some empirical advices: (1) freeze resnet backbone used for extracting 2D multi-view features. (2) keep the learning rate unchanged especially if your camera setting differs significantly from nuScenes.

**Comment by booker-max** [comment]
Thanks, thank you for your reply and suggestions







-----原始邮件-----
发件人:EKILI ***@***.***>
发送时间:2025-03-14 09:44:00 (星期五)
收件人: WU-CVGL/Omni-Scene ***@***.***>
抄送: booker-max ***@***.***>, Author ***@***.***>
主题: Re: [WU-CVGL/Omni-Scene] Great job, some questions about the dataset and running demo. (Issue #4)



First and foremost, I want to express my sincere admiration for your excellent work. The way you ingeniously combined pixel-tracing and volume-based methods is truly impressive.

I have several questions regarding your work that I would greatly appreciate your insights on:

Could you please share the approximate size of the pre-processed dataset? This information would help me better plan my download strategy.
If I intend to use this as a tool or run demonstrations, would it be sufficient to download only the dataset_omniscene_part00 package? Or do you have any alternative suggestions that would allow me to experiment with and validate its effectiveness more efficiently?
Additionally, I'm planning to fine-tune the model using my own autonomous driving dataset. In this regard, I would greatly value your recommendations on:
Which modules should be frozen
Which modules should be trained with a lower learning rate
Any other best practices you might suggest for the fine-tuning process

Thank you very much for your time and consideration. I look forward to your response.

Best regards

Thanks for your admiration! About your concerns:

The approximate size of the pre-processed dataset is 600GB. You can download it via Onedrive or BaiduYunPan.
Thanks! You really inspired me! I will provide a small subset of our pre-processed data that is easy to download for convenient demo running. But for now, you can't only download dataset_omniscene_part00 to extract files validly.
I have not tried this before. But I can give some empirical advices: (1) freeze resnet backbone used for extracting 2D multi-view features. (2) keep the learning rate unchanged especially if your camera setting differs significantly from nuScenes.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

wswdx left a comment (WU-CVGL/Omni-Scene#4)

First and foremost, I want to express my sincere admiration for your excellent work. The way you ingeniously combined pixel-tracing and volume-based methods is truly impressive.

I have several questions regarding your work that I would greatly appreciate your insights on:

Could you please share the approximate size of the pre-processed dataset? This information would help me better plan my download strategy.
If I intend to use this as a tool or run demonstrations, would it be sufficient to download only the dataset_omniscene_part00 package? Or do you have any alternative suggestions that would allow me to experiment with and validate its effectiveness more efficiently?
Additionally, I'm planning to fine-tune the model using my own autonomous driving dataset. In this regard, I would greatly value your recommendations on:
Which modules should be frozen
Which modules should be trained with a lower learning rate
Any other best practices you might suggest for the fine-tuning process

Thank you very much for your time and consideration. I look forward to your response.

Best regards

Thanks for your admiration! About your concerns:

The approximate size of the pre-processed dataset is 600GB. You can download it via Onedrive or BaiduYunPan.
Thanks! You really inspired me! I will provide a small subset of our pre-processed data that is easy to download for convenient demo running. But for now, you can't only download dataset_omniscene_part00 to extract files validly.
I have not tried this before. But I can give some empirical advices: (1) freeze resnet backbone used for extracting 2D multi-view features. (2) keep the learning rate unchanged especially if your camera setting differs significantly from nuScenes.

**Comment by wswdx** [comment]
> First and foremost, I want to express my sincere admiration for your excellent work. The way you ingeniously combined pixel-tracing and volume-based methods is truly impressive.
> 
> I have several questions regarding your work that I would greatly appreciate your insights on:
> 
> 1. Could you please share the approximate size of the pre-processed dataset? This information would help me better plan my download strategy.
> 2. If I intend to use this as a tool or run demonstrations, would it be sufficient to download only the dataset_omniscene_part00 package? Or do you have any alternative suggestions that would allow me to experiment with and validate its effectiveness more efficiently?
> 3. Additionally, I'm planning to fine-tune the model using my own autonomous driving dataset. In this regard, I would greatly value your recommendations on:
>    Which modules should be frozen
>    Which modules should be trained with a lower learning rate
>    Any other best practices you might suggest for the fine-tuning process
> 
> Thank you very much for your time and consideration. I look forward to your response.
> 
> Best regards

Hi there! I've just upload a small subset (300MB) of our full data for your convenience in running demo and checking data.

**Comment by booker-max** [comment]
Thanks.
